src/Aspire.Dashboard/Components/Layout/MainLayout.razor

```diff
@@ -28,6 +28,9 @@
     <FluentBodyContent Class="custom-body-content">
         @Body
     </FluentBodyContent>
+    <FluentFooter>
```

Comment:
Could we add another footer message that is left aligned? With the following:

`Powered by @(System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription)`

Reply:
Done. Also removed the padding on the right so the scrollbar attaches to the side of the window.
 
![image](https://github.com/dotnet/aspire/assets/1761079/9a00d207-c542-4853-b503-8fddbb8eb2d0)